main.go

```diff
@@ -69,9 +69,10 @@ var cmds = []*cli.Command{
 				Usage: "Source for upgrade. Composed of `type:address`. Accepts `file:`,`dir:` or `oci:` for the type of source.\nFor example `file:/var/share/myimage.tar`, `dir:/tmp/extracted` or `oci:repo/image:tag`",
 			},
 			&cli.BoolFlag{Name: "pre", Usage: "Include pre-releases (rc, beta, alpha)"},
+			&cli.BoolFlag{Name: "recovery", Usage: "Upgrade recovery"},
 		},
 		Description: `
-Manually upgrade a kairos node.
+Manually upgrade a kairos node Active image. Does not upgrade passive or recovery images.
```

Comment:
well, it does if you pass `recovery` right? Maybe it should be more like:
`By default, it does not upgrade passive or recovery images` ?